rhoas-cli-kafka/README.adoc

```diff
@@ -0,0 +1,265 @@
+////
+START GENERATED ATTRIBUTES
+WARNING: This content is generated by running npm --prefix .build run generate:attributes
+////
+
+
+:community:
+:imagesdir: ./images
+:product-version: 1
+:product-long: Application Services
+:product: App Services
+:registry-product-long: OpenShift Service Registry
+:registry: Service Registry
+// Placeholder URL, when we get a HOST UI for the service we can put it here properly
+:service-url: https://console.redhat.com/beta/application-services/streams/
+:registry-url: https://console.redhat.com/beta/application-services/service-registry/
+:property-file-name: app-services.properties
+:rhoas-version: 0.29.0
+
+// Other upstream project names
+:samples-git-repo: https://github.com/redhat-developer/app-services-guides
+
+//URL components for cross refs
+:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/
+:base-url-cli: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
+:getting-started-url: getting-started/README.adoc
+:kafka-bin-scripts-url: kafka-bin-scripts/README.adoc
+:kafkacat-url: kafkacat/README.adoc
+:quarkus-url: quarkus/README.adoc
+:rhoas-cli-url: rhoas-cli/README.adoc
+:rhoas-cli-ref-url: commands
+:topic-config-url: topic-configuration/README.adoc
+:consumer-config-url: consumer-configuration/README.adoc
+:service-binding-url: service-discovery/README.adoc
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-getting-started-rhoas-cli"]
+= Getting started with the rhoas CLI and RHOSAK (Kafka)
+:context: getting-started-rhosak
+
+[IMPORTANT]
+====
+The `rhoas` command-line interface (CLI) is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
+====
+
```

Comment:
With the `rhoas` command-line interface (CLI), you can manage your application services from a terminal. 

This guide describes how to get started quickly by using the `rhoas` CLI to create a simple Kafka instance, connecting to it, and creating a Kafka topic to produce and consume messages.

Reply:
To do the three-way split, we need an install prereq. 

.Prerequisites
* You have downloaded and installed the latest version of the 'rhoas' CLI.
+
For instructions on installing the `rhoas` cli, see  information, see {base-url}{rhoas-cli-url}[_Installing the rhoas CLI _^]

Reply:
Do you mind adding this directly to make sure that we got this right